packages/components/src/box-control/stories/index.story.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import BoxControl from '../';
+import type { BoxControlValue } from '../types';
+
+const meta: Meta< typeof BoxControl > = {
+	title: 'Components (Experimental)/BoxControl',
+	component: BoxControl,
+	argTypes: {
+		onChange: { action: 'onChange' },
+	},
+	parameters: {
+		controls: { expanded: true },
+		docs: { canvas: { sourceState: 'shown' } },
+	},
+};
+export default meta;
+
+const defaultSideValues = {
+	top: '10px',
+	right: '10px',
+	bottom: '10px',
+	left: '10px',
+};
+
+const Template: StoryFn< typeof BoxControl > = ( props ) => {
+	const [ values, setValues ] =
+		useState< BoxControlValue >( defaultSideValues );
+
+	return (
+		<BoxControl
+			label="Padding"
```

Comment:
As a convention, we should always try to use the story `args` to set the value of the props, instead of hardcoding them on the component in the template. 

For example, for label, we could do something like

```diff
diff --git a/packages/components/src/box-control/stories/index.story.tsx b/packages/components/src/box-control/stories/index.story.tsx
index 3186ba23c5..3b922f26b3 100644
--- a/packages/components/src/box-control/stories/index.story.tsx
+++ b/packages/components/src/box-control/stories/index.story.tsx
@@ -40,7 +40,6 @@ const Template: StoryFn< typeof BoxControl > = ( props ) => {
 
 	return (
 		<BoxControl
-			label="Padding"
 			values={ values }
 			{ ...props }
 			onChange={ ( nextValue ) => setValues( nextValue ) }
@@ -50,27 +49,31 @@ const Template: StoryFn< typeof BoxControl > = ( props ) => {
 
 export const Default = Template.bind( {} );
 Default.args = {
-	label: 'Box Control',
 	values: undefined,
+	label: 'Label',
 };
 
 export const ArbitrarySides = Template.bind( {} );
 ArbitrarySides.args = {
+	...Default.args,
 	sides: [ 'top', 'bottom' ],
 };
 
 export const SingleSide = Template.bind( {} );
 SingleSide.args = {
+	...Default.args,
 	sides: [ 'bottom' ],
 };
 
 export const AxialControls = Template.bind( {} );
 AxialControls.args = {
+	...Default.args,
 	splitOnAxis: true,
 };
 
 export const AxialControlsWithSingleSide = Template.bind( {} );
 AxialControlsWithSingleSide.args = {
+	...Default.args,
 	sides: [ 'horizontal' ],
 	splitOnAxis: true,
 };

```